packages/graphback-core/src/relationships/relationshipHelpers.ts

```diff
@@ -0,0 +1,155 @@
+import { parseMarker } from 'graphql-metadata';
```

Comment:
This place is kinda evil. I will at least namespace those fields.
Any reason why we cannot have this as part of the metadata implementation so we can get that enclosed and available thru single object on model rather than allowing plugin creators to use them directly.